### PR TITLE
Refactor/draft decisions

### PR DESCRIPTION
**Pull request by nvdk**
This PR refactors the draft decision routes to be octane compatible. I've also taken the liberty to move some useful old stuff from editor-documents and removed the old editor-documents related code. 

As I've changed a lot of stuff, this will likely have many bugs and needs thorough testing before we can merge it :(.

### CODE REVIEW

File: app/templates/agendapoints/edit.hbs
Comment (abeforgit) [inline app/templates/agendapoints/edit.hbs:4]: you could use the perform helper here, which would mean you can get rid of the this.save method in the js file
`hash action=(perform this.saveTask) isRunning=this.saveTask.isRunning`

**Review by abeforgit — APPROVED** [review]
Not sure if this is because of your changes, but when restoring an agendapoint from the trash, it gets restored but the ui isnt immediately updated (only after refresh)

**Comment by abeforgit** [comment]
ugh didnt mean to approve

but its a small thing anyway

**Comment by nvdk** [comment]
the ui not updating was already an issue, which I think I fixed in the other PR. I'll add the perform helper, makes sense :).

**Comment by nvdk** [comment]
I'll wait for another review to merge this, as I think it can use some extra eyes :)

**Comment by Asergey91** [comment]
sorry I didn't post here earlier.

so after the latest update to this branch most of the issues I had seemed to go away

however there is still some wierdness when you create a document and try to name it. I think its just an issue with preview text vs default text

**Comment by Asergey91** [comment]
most of the issues have been solved for me, however there is an issue in the agendapoint component i discovered while testing this. https://binnenland.atlassian.net/browse/GN-2350

**Comment by Asergey91** [comment]
most of the issues have been solved for me, however there is an issue in the agendapoint component i discovered while testing this. https://binnenland.atlassian.net/browse/GN-2350

**Review by Asergey91 — APPROVED** [review]

**Pull request merged**
